tests/test_handlers/test_ome_zarr.py

```diff
@@ -15,17 +15,28 @@
 else:
     zarr = pytest.importorskip("zarr")
 
-
-@pytest.mark.parametrize("store", ["out.zarr", None, "tmp"])
-def test_ome_zarr_writer(store: str | None, tmp_path: Path, core: CMMCorePlus) -> None:
-    mda = useq.MDASequence(
-        channels=["Cy5", "FITC"],
-        time_plan={"interval": 0.1, "loops": 3},
-        stage_positions=[(222, 1, 1), (111, 0, 0)],
-        z_plan={"range": 0.3, "step": 0.1},
-        axis_order="tpcz",
-    )
-
+full_mda = useq.MDASequence(
+    channels=["Cy5", "FITC"],
+    time_plan={"interval": 0.1, "loops": 3},
+    stage_positions=[(222, 1, 1), (111, 0, 0)],
+    z_plan={"range": 0.3, "step": 0.1},
+    axis_order="tpcz",
+)
+
+part_mda = useq.MDASequence(
+    channels=["Cy5", "FITC"],
+    stage_positions=[(222, 1, 1), (111, 0, 0)],
+    time_plan={"interval": 0.1, "loops": 3},
+)
```

Comment:
```suggestion
part_mda = useq.MDASequence(
     channels=["Cy5", "FITC"],
     stage_positions=[(222, 1, 1), (111, 0, 0)],
     time_plan={"interval": 0.1, "loops": 3},
 )
 
 full_mda = part_mda.replace(axis_order="tpcz", channels=["Cy5", "FITC"])
```

just a nice syntax to extend or build off of another MDA